### ISSUE TITLE
Make this H1 and add alt text

### ISSUE THREAD

**Issue by ciaoaaron** [caplank-1/cPlank_SinglePageLayout#2]
https://github.com/caplank-1/cPlank_SinglePageLayout/blob/848ca64ec31897862aee9554ae11f00a0efb12fc/index.html#L10

alt text should be the name of this site.
